docs/source/java/vector_schema_root.rst

```diff
@@ -15,9 +15,6 @@
 .. specific language governing permissions and limitations
 .. under the License.
 
-.. default-domain:: java
-.. highlight:: java
-
```

Comment:
Did you mean to remove this?

Reply:
Yes, this cause error like this: `/arrow/docs/source/java/vector_schema_root.rst:81: ERROR: Unknown interpreted text role "class".`